players/rikku.py

```diff
@@ -0,0 +1,111 @@
+from players.base import Player
+import logging
+import memory
+import xbox
+import battle
+
+logger = logging.getLogger(__name__)
+
+class RikkuImp(Player):
+    
+    def __init__(self):
+        super().__init__("Rikku", 6, [0, 20, 1])
+        
+        
+    def rikku_od_items(self, slot):
+        _navigate_to_position(slot, battle_cursor=memory.main.rikku_od_cursor_1)
+
+
+    def overdrive(battle):
+        # First, determine which items we are using
+        if battle == "tutorial":
+            item1 = memory.main.get_item_slot(73)
+            logger.debug(f"Ability sphere in slot: {item1}")
+            item2 = item1
+        elif battle == "Evrae":
+            if game_vars.skip_kilika_luck():
+                item1 = memory.main.get_item_slot(81)
+                logger.debug(f"Lv1 sphere in slot: {item1}")
+                item2 = memory.main.get_item_slot(84)
+                logger.debug(f"Lv4 sphere in slot: {item2}")
+            else:
+                item1 = memory.main.get_item_slot(94)
+                logger.debug(f"Luck sphere in slot: {item1}")
+                item2 = memory.main.get_item_slot(100)
+                logger.debug(f"Map in slot: {item2}")
+        elif battle == "Flux":
+            item1 = memory.main.get_item_slot(35)
+            logger.debug(f"Grenade in slot: {item1}")
+            item2 = memory.main.get_item_slot(85)
+            logger.debug(f"HP Sphere in slot: {item2}")
+        elif battle == "trio":
+            item1 = 108
+            item2 = 108
+            logger.debug(f"Wings are in slot: {item1}")
+        elif battle == "crawler":
+            item1 = memory.main.get_item_slot(30)
+            logger.debug(f"Lightning Marble in slot: {item1}")
+            item2 = memory.main.get_item_slot(85)
+            logger.debug(f"Mdef Sphere in slot: {item2}")
+        elif battle == "spherimorph1":
+            item1 = memory.main.get_item_slot(24)
```

Comment:
As a future cleanup, adding a `mix` method that takes two items would help clean this up a bunch.

Reply:
Agreed.